lib/tasks/webpacker/compile.rake

```diff
@@ -1,6 +1,7 @@
 $stdout.sync = true
 
 require "open3"
+require "rbconfig"
```

Comment:
Hey, don't think we need to require `rbconfig` here. Does it not work on windows without this?